functions.opengraph.php

```diff
@@ -322,9 +322,18 @@ function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) {
 		$blavatar_domain = blavatar_domain( site_url() );
 		if ( blavatar_exists( $blavatar_domain ) ) {
 			$image_url = blavatar_url( $blavatar_domain, 'img', $width, false, true );
-			$image_size = getimagesize( $image_url );
-			$img_width = $image_size[0];
-			$img_height = $image_size[1];
+
+			$img_width  = '';
+			$img_height = '';
+			$image_id = attachment_url_to_postid( $image_url );
```

Comment:
Since the code in this PR is already merged and released, I opened #6061 so we don't lose your comments. Let's continue the conversation there.